server/portal/apps/projects/managers/base.py

```diff
@@ -181,6 +181,32 @@ def list(self, offset=0, limit=100):
             limit=limit
         )]
 
+    def apply_permissions(self, project, username, acl=None):
+        """Index project and update acls
+        """
+        project_id = project.project_id
+        project_indexer.apply_async(args=[project_id])
```

Comment:
Do we need to index a project on permissions change? Would that affect the results of a search at all?

Reply:
@jarosenb ?